### PR TITLE
Include Elixir 1.17/Erlang 27.0 in CI

### PR DESCRIPTION
**Pull request by kianmeng**

### CODE REVIEW

File: mix.lock [hunk 1]
```diff
@@ -2,7 +2,7 @@
...
   "connection": {:hex, :connection, "1.1.0", "ff2a49c4b75b6fb3e674bfc5536451607270aac754ffd1bdfe175abe4a6d7a68", [:mix], [], "hexpm", "722c1eb0a418fbe91ba7bd59a47e28008a189d47e37e0e7bb85585a016b2869c"},
-  "credo": {:hex, :credo, "1.7.3", "05bb11eaf2f2b8db370ecaa6a6bda2ec49b2acd5e0418bc106b73b07128c0436", [:mix], [{:bunt, "~> 0.2.1 or ~> 1.0", [hex: :bunt, repo: "hexpm", optional: false]}, {:file_system, "~> 0.2 or ~> 1.0", [hex: :file_system, repo: "hexpm", optional: false]}, {:jason, "~> 1.0", [hex: :jason, repo: "hexpm", optional: false]}], "hexpm", "35ea675a094c934c22fb1dca3696f3c31f2728ae6ef5a53b5d648c11180a4535"},
+  "credo": {:hex, :credo, "1.7.7", "771445037228f763f9b2afd612b6aa2fd8e28432a95dbbc60d8e03ce71ba4446", [:mix], [{:bunt, "~> 0.2.1 or ~> 1.0", [hex: :bunt, repo: "hexpm", optional: false]}, {:file_system, "~> 0.2 or ~> 1.0", [hex: :file_system, repo: "hexpm", optional: false]}, {:jason, "~> 1.0", [hex: :jason, repo: "hexpm", optional: false]}], "hexpm", "8bc87496c9aaacdc3f90f01b7b0582467b69b4bd2441fe8aae3109d843cc2f2e"},
```
Comment (kianmeng) [inline mix.lock:5]: Updates needed for Credo:

```
Checking 21 source files ...
Error while running Elixir.Credo.Check.Readability.LargeNumbers on lib/fun_with_flags.ex
Error while running Elixir.Credo.Check.Readability.LargeNumbers on lib/fun_with_flags/flag.ex
Error while running Elixir.Credo.Check.Readability.LargeNumbers on lib/fun_with_flags/application.ex
Error while running Elixir.Credo.Check.Readability.LargeNumbers on lib/fun_with_flags/config.ex
Error while running Elixir.Credo.Check.Readability.MaxLineLength on lib/fun_with_flags/application.ex
Error while running Elixir.Credo.Check.Readability.MaxLineLength on lib/fun_with_flags/config.ex
Error while running Elixir.Credo.Check.Readability.MaxLineLength on lib/fun_with_flags/flag.ex
Error while running Elixir.Credo.Check.Readability.MaxLineLength on lib/fun_with_flags.ex

Error: 17:09:43.219 [error] Task #PID<0.6387.0> started from #PID<0.6313.0> terminating
** (CaseClauseError) no case clause matching: {:ok, 39, 1, [], [{:eol, {38, 4, 1}}, {:end, {38, 1, nil}}, {:eol, {37, 6, 1}}, {:end, {37, 3, nil}}, {:eol, {36, 43, 1}}, {:")", {36, 42, nil}}, {:"(", {36, 41, nil}}, {:paren_identifier, {36, 27, ~c"all_flag_names"}, :all_flag_names}, {:., {36, 26, nil}}, {:")", {36, 25, nil}}, {:"(", {36, 24, nil}}, {:paren_identifier, {36, 5, ~c"persistence_adapter"}, :persistence_adapter}, {:eol, {35, 24, 1}}, {:do, {35, 22, nil}}, {:do_identifier, {35, 7, ~c"all_flag_names"}, :all_flag_names}, {:identifier, {35, 3, ~c"def"}, :def}, {:eol, {34, 60, 1}}, {:"}", {34, 59, nil}}, {:")", {34, 58, nil}}, {:"(", {34, 57, nil}}, {:paren_identifier, {34, 54, ~c"any"}, :any}, {:",", {34, 52, 0}}, {:atom, {34, 46, ~c"error"}, :error}, {:"{", {34, 45, nil}}, {:pipe_op, {34, 43, nil}, :|}, {:"}", {34, 41, nil}}, {:"]", {34, 40, nil}}, {:identifier, {34, 36, ~c"atom"}, :atom}, {:"[", {34, 35, nil}}, {:",", {34, 33, 0}}, {:atom, {34, 30, ~c"ok"}, :ok}, {:"{", {34, 29, nil}}, {:type_op, {34, 26, nil}, :"::"}, {:")", {34, 24, nil}}, {:"(", {34, 23, nil}}, {:paren_identifier, {34, 9, ~c"all_flag_names"}, :all_flag_names}, {:identifier, {34, 4, ~c"spec"}, :spec}, {:at_op, {34, 3, nil}, :@}, {:eol, {32, 6, 2}}, {:end, {32, 3, nil}}, {:eol, {31, 38, ...}}, {:")", {31, ...}}, {:"(", {...}}, {:paren_identifier, ...}, {...}, ...], []}
** (EXIT from #PID<0.95.0>) an exception was raised:
```

**Comment by tompave** [comment]
Hi, thank you for the contribution!
And apologies, but yesterday I started catching up with some long overdue maintenance on this project, and the first thing I did was updating the CI setup. I did that in https://github.com/tompave/fun_with_flags/pull/179, before seeing your PR.

This is now already done, therefore I'll close it.

**Comment by kianmeng** [comment]
> Hi, thank you for the contribution! And apologies, but yesterday I started catching up with some long overdue maintenance on this project, and the first thing I did was updating the CI setup. I did that in #179, before seeing your PR.
> 
> This is now already done, therefore I'll close it.

Noted, and thanks for this library and keeping it up-to-date!

I noticed there is a V2 branch, https://github.com/tompave/fun_with_flags/tree/v2, are we going to see a V2 release in foreseeable future?

**Comment by tompave** [comment]
Thank you for the kind words.

> I noticed there is a V2 branch, https://github.com/tompave/fun_with_flags/tree/v2, are we going to see a V2 release in foreseeable future?

I've been working on it on and off, but I have no clear timeline in mind.

**Pull request closed without merging**